schema/shops.ddl

```diff
@@ -1,11 +1,11 @@
 CREATE TABLE `Shops` (
   `ShopID` BIGINT NOT NULL,
-  `MerchantID` BIGINT NOT NULL,
+  `MerchantID` VARCHAR(30) NOT NULL,
```

Comment:
How did you confirm that this is only 30 characters long?

Reply:
@onish-garg

Reply:
Via circumstantial evidence for most part. Both Diksha and I have 29 character long `sub` (unique ID provided by Identity API). We tried to look around for more info about its length but could not really find anything, so we settled for it then. However, I can see how this can cause issues. How do we go about this?

Reply:
Via circumstantial evidence for most part. Both Diksha and I have 29 character `sub` (unique ID provided by Identity API). We tried to look around for more info about its length but could not really find anything, so I settled for it then. However, I can see how this can cause issues. How do we go about this?

Reply:
`Maximum length of 255 case-sensitive ASCII characters.` https://developers.google.com/identity/protocols/oauth2/openid-connect#an-id-tokens-payload
@dikshamodi could you please update the `VARCHAR` size?

Reply:
Done